envoy.base.utils/envoy/base/utils/abstract/project/changelog.py

```diff
@@ -141,7 +153,9 @@ def get_data(cls, path) -> typing.ChangelogDict:
             {k: (v
                  if k == "date"
                  else [dict(area=c["area"],
-                            change=typing.Change(c["change"]))
+                            change=typing.Change(c["change"]),
+                            **({"prs": c["prs"]} if "prs" in c else
+                               {"prs": [c["pr"]]} if "pr" in c else {}))
```

Comment:
this should just use something like `c.get("prs", {})`

Reply:
Done, Thanks! :)